proc-macros/src/api_def.rs

```diff
@@ -218,3 +217,29 @@ impl syn::parse::Parse for ApiMethods {
 		Ok(out)
 	}
 }
+
+#[cfg(test)]
+mod tests {
+	use super::*;
+
+	#[test]
+	fn returns_ok_if_attribue_does_not_exists() {
+		let mut a = ApiMethodAttrs { method: None, positional_params: true };
+		let b = ApiMethodAttrs { method: Some("method".to_string()), positional_params: true };
+		assert!(ApiMethodAttrs::try_merge(&mut a, b).is_ok());
+	}
+
+	#[test]
+	fn returns_an_error_if_attribue_already_exists() {
```

Comment:
```suggestion
	fn returns_an_error_if_attribute_already_exists() {
```

Reply:
The test name speaks of returning an error, but the assertion does not check for that. Which is it?

Reply:
🤦 good catch (+ the damn typos).